data/protoc

```diff
@@ -0,0 +1 @@
+https://github.com/google/protobuf/releases/download/v3.2.0/protoc-3.2.0-linux-x86_64.zip
```

Comment:
nit: it might be good to name this file as protoc_url.txt so that it reflects the content when people are just scanning the directory list.  Otherwise, someone might assume this is the actual protoc binary by the file name.